### ISSUE TITLE
Netboot with NixOS

### ISSUE THREAD

**Issue by gianarb** [danderson/netboot#136]
Hello! Awesome project. Simple and it does what it should I love it!

I am writing this question because I think you are faimiliar with NixOS.
I am following this guide to netboot a NixOS based operating system

https://nixos.wiki/wiki/Netboot

It generates a ipxe script file that contains the init cmdline

```
$ cat result/netboot.ipxe
#!ipxe
# Use the cmdline variable to allow the user to specify custom kernel params
# when chainloading this script from other iPXE scripts like netboot.xyz
kernel bzImage init=/nix/store/wc9fvfyhv649b4kxry5747rx5rw88v68-nixos-system-nixos-22.05.20220902.67e4507/init initrd=initrd loglevel=4 ${cmdline}
initrd initrd
boot
```

I am trying to figure out a way to teach pixiecore about this file or at least about the init location, ideally via [services.pixiecore.cmdLine](https://search.nixos.org/options?channel=22.05&show=services.pixiecore.cmdLine&from=0&size=50&sort=relevance&type=packages&query=pixiecore)

TLDR: I am not sure about how to glue all of this together in a good way
